src/App.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import CountButton from './CountButton';
+import NumberButtons from './NumberButtons';
```

Comment:
```suggestion
import { useState } from 'react';

import CountButton from './CountButton';
import NumberButtons from './NumberButtons';
```

의미상으로 구분되는 경우 빈 줄어 넣어서 구분해주세요.
여기서는 범용 라이브러리 와 컴포넌트들로 구분할 수 있을 것 같아요.
코드를 작성할때는 빈 줄 하나에도 의미를 담아서 작성하신다 생각하시면 좋아요 ;)

Reply:
넵넵 앞으로도 구분하도록 하겠습니다!